<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPrint</class>
 <widget class="QDialog" name="dlgPrint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export PDF / Print</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../.designer/backup/Images/Icons/Print.png</normaloff>../../../../../.designer/backup/Images/Icons/Print.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <widget class="QCheckBox" name="chboxIDExtra">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Additional Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeader">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Select the fields you want to include:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chboxID">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Basic Information</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chboxAddress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chboxPersoData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>163</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Personal Notes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCancel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPrint">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Print...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbGeneratePdf">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>108</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Create PDF...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chboxSignature">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Signed PDF document</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHeader_Options">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Options:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chboxID</tabstop>
  <tabstop>chboxIDExtra</tabstop>
  <tabstop>chboxAddress</tabstop>
  <tabstop>chboxPersoData</tabstop>
  <tabstop>chboxSignature</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbPrint</tabstop>
  <tabstop>pbGeneratePdf</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
